compilers/concrete-compiler/compiler/lib/Dialect/FHELinalg/Transforms/Tiling.cpp

```diff
@@ -186,9 +186,25 @@ class FHELinalgTilingMarkerPass
         mlir::Builder(&this->getContext()).getI64ArrayAttr(tileSizes);
 
     op->walk([&](mlir::Operation *op) {
-      if (llvm::isa<mlir::concretelang::FHELinalg::ApplyLookupTableEintOp>(
+      if (llvm::isa<mlir::concretelang::FHELinalg::ApplyLookupTableEintOp,
```

Comment:
why use or? and not have just one llvm::isa? should be an interface?

Reply:
The idea is to group similar operations into each instantiation of `llvm::isa`. Compared to a single instantiation, this should not result in a performance hit, since the variadic template is recursively replaced with instantiations of the scalar version of `llvm::isa` reduced with the `||` operator.

I think it's not worth adding an op interface (or rather trait) for this, since there is only this one place where the property is checked.

However, if you insist, I am happy to either merge the calls into a single instantiation of `llvm::isa` or to add an op interface.